app/models/vm_or_template.rb

```diff
@@ -1193,7 +1193,7 @@ def refresh_on_reconfig
     unless ext_management_system.authentication_status_ok?
       raise _("%{table} failed last authentication check") % {:table => ui_lookup(:table => "ext_management_systems")}
     end
-    EmsRefresh.reconfig_refresh(self)
+    EmsRefresh.refresh(self)
```

Comment:
@jameswnl wasn't the idea that we could call `queue_refresh` so that the `Refresh Worker` would handle the refresh, but add the `sync` parameter so we were sure that the inventory was updated before making the policy call?